README.md

```diff
@@ -141,6 +141,11 @@ target_link_libraries(myprogram PUBLIC fast_float)
 
 ```
 
+## Using as single header
+
+The script `amalgamate.py` may be used to generate a single header version of the library is so desired.
```

Comment:
maybe we want to write `script/amalgamate.py`.

Reply:
Make sense.